nrt/data/DynaFlow/GeneratorPVTfo/GeneratorPVTfo.crv

```diff
@@ -46,6 +46,8 @@
   <curve model="Generator5" variable="reactivePowerControlLoop_level"/>
   <curve model="Generator5" variable="reactivePowerControlLoop_QStatorPu"/>
   <curve model="Generator5" variable="reactivePowerControlLoop_UStatorRefPu"/>
+  <curve model="Generator5" variable="reactivePowerControlLoop_integrator_y"/>
```

Comment:
As you have changed the curves in the nrt, you should also change the reference csv.
Could you also test the nrt ? Thanks

Reply:
I only added this curve for debugging. Now it is removed.
nrt test : done